R/process_sc_all_sds.R

```diff
@@ -59,14 +62,19 @@ process_sc_all_sds <- function(
       sds_end_date = fix_sc_end_dates(
         .data$sds_start_date,
         .data$sds_end_date,
-        .data$period
+        .data$sds_period_end_date
       )
     ) %>%
+    dplyr::select(
+      -sds_period_start_date, -sds_period_end_date,
+      -sds_start_date_after_end_date
+    ) %>%
     # rename for matching source variables
     dplyr::rename(
       record_keydate1 = .data$sds_start_date,
       record_keydate2 = .data$sds_end_date
     ) %>%
+    dplyr::distinct() %>%
```

Comment:
Im confused here why we need distinct again as we do this on line 88

Reply:
I just love distinct. If I've done a lot of mutates I tend to do a distinct to get rid of any duplicates that mutate might have introduced. Not sure I need as many as I've added.... I just really love the distinct function